config/filter/config.json

```diff
@@ -21,5 +21,11 @@
     "IMU_accel_threshold": 0.5,
 
     "RefCoords": {"lat": 42.2766, "long": -83.7382},
```

Comment:
are these the approx coordinates of ann arbor rn?

Reply:
yep, ig a sidenote is we need to remember to change this for URC and CIRC

Reply:
what should they be changed to for each respective comp?

Reply:
I think any GPS point within a few miles of the comp should be fine